dashboard/test/controllers/census_reviewers_controller_test.rb

```diff
@@ -0,0 +1,115 @@
+require 'test_helper'
+
+class CensusReviewersControllerTest < ActionController::TestCase
+  setup do
+    @reviewer = create :census_reviewer
+  end
+
+  # Add tests that you redirect to sign in, get a 403 if don't have perms.
```

Comment:
is this comment a todo?

Reply:
It was a todo that I missed. I've added those tests now.